strongbox-web-core/src/main/java/org/carlspring/strongbox/config/WebConfig.java

```diff
@@ -187,6 +195,22 @@ public Validator localValidatorFactoryBean()
     {
         return new LocalValidatorFactoryBean();
     }
+    
+    @Bean
+    @Qualifier("loggingManagementDirectoryListingService")
+    public DirectoryListingService getLoggingManagementDirectoryListingService()
+    {
+        String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
+        return new DirectoryListingServiceImpl(String.format("%s/api/logging", baseUrl));
+    }
+
+    @Bean
+    @Qualifier("browseRepositoryDirectoryListingService")
+    public DirectoryListingService getBrowseRepositoryDirectoryListingService()
+    {
+        String baseUrl = StringUtils.chomp(configurationManager.getConfiguration().getBaseUrl(), "/");
```

Comment:
both methods `getLoggingManagementDirectoryListingService()` and `getBrowseRepositoryDirectoryListingService()` differ only by the path pattern, so almost 2 lines are code duplication and could be extracted to one method with `String` argument